### ISSUE TITLE
Add Cookie3 analytics

### ISSUE THREAD

**Issue by peterpolman** [thxprotocol/monorepo-legacy#1061]

**Comment by peterpolman** [comment]
@miesz-ko Can you provide docs and/or the script that needs to be added?

**Comment by miesz-ko** [comment]
> @miesz-ko Can you provide docs and/or the script that needs to be added?

Yeah, the problem here is that the tooling is not very affordable. Let's move this one to the next sprint and let me negotiate one more time.

**Comment by peterpolman** [comment]
@miesz-ko Let me know what to do with this one based on that meeting you had in todays afternoon!
